htmresearch/encoders/__init__.py

```diff
@@ -22,8 +22,15 @@
 from enum import Enum
 
 
-
+# TODO: this should be deleted (or at least moved to cio_encoder.py)
```

Comment:
Yes, better in cio_encoder.py b/c word and document specify the Cio /terms and /text endpoints, respectively.